Components/Avatar/avatar.css

```diff
@@ -0,0 +1,33 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+:root {
+    --avatar-size: 2em;
+    --avatar-border: none;
+}
```

Comment:
You can create a separate variable style.

Reply:
A separate css file for all right?

Reply:
Yes, a separate CSS file for all varaibles.

Reply:
Ok, actually I did not do that because if you visit my live preview, you can see I am also providing individual component CSS imports, so if anyone wants to use only badge component, why should he import CSS variables of other files, that's why put them in their respective CSS files.